### DISCUSSION TITLE
Light mode is broken

### DISCUSSION THREAD

**Discussion by McKpol** [TomSchimansky/CustomTkinter#1866] [General]
![image](https://github.com/TomSchimansky/CustomTkinter/assets/104125769/db0942dd-3b7f-4180-971e-a9196b93f5a9)

There is project: https://github.com/McKpol/CustomRichPresenceInPython/tree/CustomRichPresenceInPython

**Comment by McKpol** [comment]
Maybe is just error in my code idk
Is set to Light but is thinking is dark so yeah...

**Comment by avalon60** [comment]
Your  change_appearance_mode_event callback function expects a new_apperance_mode parameter, which you don't pass. That said, it should be able to do a get() on the widget value, to determine what to set the appearance mode to, so it doesn't need a parameter.

**Comment by McKpol** [comment]
Yeah. But settings (TopLevel) works fine and not the main App.
to set up the apperance mode to main used only self._set_appearance_mode(apperancemode)
which is correct and its look like it not my fault.

**Comment by ElectricCandlelight** [comment]
Change line 153 in your code to ``` customtkinter.set_appearance_mode(apperancemode)```
